src/cdk/v2/destinations/bluecore/utils.js

```diff
@@ -143,16 +146,48 @@ const addProductArray = (products) => {
   return finalProductArray;
 };
 
+function isObject(item) {
+  return item && typeof item === 'object' && !Array.isArray(item);
+}
+
+/**
+ * Recursively merges multiple objects into a single object.
+ *
+ * @param {Object} target - The target object to merge into.
+ * @param {...Object} sources - The source objects to merge from.
+ * @returns {Object} - The merged object.
+ */
+function deepMerge(target, ...sources) {
```

Comment:
don't we have any lodash function for this ?

Reply:
added